stistools/ocrreject.py

```diff
@@ -268,10 +268,10 @@ def ocrreject(input, output,
     if crradius:
         optional_args.append("-radius")
         optional_args.append("%.10g" % crradius)
-    if crthresh:
+    if crthresh is not None:
         optional_args.append("-thresh")
         optional_args.append("%.10g" % crthresh)
```

Comment:
Maybe 
optional_args.append(f"{crthresh: .10g})
is a bit more current style

Reply:
I updated the code to use f-strings.